apps/wallet/src/features/Send/InputTransaction/index.tsx

```diff
@@ -30,6 +34,16 @@ const InputTransaction: FC<Props> = ({ navigator }) => {
 		}
 	};
 
+	useEffect(() => {
+		if (collectible) {
+			txActions.update({ type: 'Collectible' });
+			const collection = collections.find(
+				(ele) => ele._id === collectible.collectionId,
+			);
+			txActions.update({ collection });
```

Comment:
1. seems that `txActions.update` would be call just once, instead of 2 times like this
2. if the whole idea if this effect is to keep `txContext` (a valtio instance) and something managed by `txActions` the logic shouldn't be here as an effect under component level - we could e.g subscribe for changes from valtio and sync it with others in `bootstrap` or engine's setup.

Reply:
> 1. seems that `txActions.update` would be call just once, instead of 2 times like this
> 2. if the whole idea if this effect is to keep `txContext` (a valtio instance) and something managed by `txActions` the logic shouldn't be here as an effect under component level - we could e.g subscribe for changes from valtio and sync it with others in `bootstrap` or engine's setup.

1. I'll fix it
2. It would be component level because there are some specific cases that we need to check whether it's collectible tx (currently there is a case in collectible screen)